### PR TITLE
(Javadoc) Fix return tag for throwableOf*() methods

### PR DESCRIPTION
**Pull request by arend-von-reinersdorff**

### CODE REVIEW

File: src/main/java/org/apache/commons/lang3/exception/ExceptionUtils.java [hunk 1]
```diff
@@ -810,7 +810,7 @@ public static void removeCommonFrames(final List<String> causeFrames, final List
      * @param <T> the type of Throwable you are searching.
      * @param throwable  the throwable to inspect, may be null
      * @param clazz  the class to search for, subclasses do not match, null returns null
-     * @return the index into the throwable chain, null if no match or null input
+     * @return the first matching throwable form the throwable chain, null if no match or null input
```
Comment (kinow) [inline src/main/java/org/apache/commons/lang3/exception/ExceptionUtils.java:813]: s/form/from?

File: src/main/java/org/apache/commons/lang3/exception/ExceptionUtils.java [hunk 2]
```diff
@@ -835,7 +835,7 @@ public static void removeCommonFrames(final List<String> causeFrames, final List
      * @param clazz  the class to search for, subclasses do not match, null returns null
      * @param fromIndex  the (zero-based) index of the starting position,
      *  negative treated as zero, larger than chain size returns null
-     * @return the index into the throwable chain, null if no match or null input
+     * @return the first matching throwable form the throwable chain, null if no match or null input
```
Comment (kinow) [inline src/main/java/org/apache/commons/lang3/exception/ExceptionUtils.java:838]: s/form/from?

File: src/main/java/org/apache/commons/lang3/exception/ExceptionUtils.java [hunk 3]
```diff
@@ -855,7 +855,7 @@ public static void removeCommonFrames(final List<String> causeFrames, final List
      * @param <T> the type of Throwable you are searching.
      * @param throwable  the throwable to inspect, may be null
      * @param type  the type to search for, subclasses match, null returns null
-     * @return the index into the throwable chain, null if no match or null input
+     * @return the first matching throwable form the throwable chain, null if no match or null input
```
Comment (kinow) [inline src/main/java/org/apache/commons/lang3/exception/ExceptionUtils.java:858]: s/form/from?

File: src/main/java/org/apache/commons/lang3/exception/ExceptionUtils.java [hunk 4]
```diff
@@ -880,7 +880,7 @@ public static void removeCommonFrames(final List<String> causeFrames, final List
      * @param type  the type to search for, subclasses match, null returns null
      * @param fromIndex  the (zero-based) index of the starting position,
      *  negative treated as zero, larger than chain size returns null
-     * @return the index into the throwable chain, null if no match or null input
+     * @return the first matching throwable form the throwable chain, null if no match or null input
```
Comment (kinow) [inline src/main/java/org/apache/commons/lang3/exception/ExceptionUtils.java:883]: s/form/from?

File: src/main/java/org/apache/commons/lang3/exception/ExceptionUtils.java (hunk 1 above)
Comment (arend-von-reinersdorff) [reply src/main/java/org/apache/commons/lang3/exception/ExceptionUtils.java]: Hi @kinow,
fixed the typo. Thanks for spotting it :-)

**Pull request merged**